lib/db.go

```diff
@@ -0,0 +1,327 @@
+package lib
+
+import (
+	"fmt"
+
+	"github.com/goby-lang/goby/vm"
+	"github.com/goby-lang/goby/vm/errors"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+type Object = vm.Object
```

Comment:
why do we still need to export these types from db?

Reply:
@eliothedeman I still don't get the reason we export these types. is there any chance for us to use `lib.Object` or `lib.VM` instead of `vm.Object` or `vm.VM` ?